packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -59,7 +59,7 @@ export interface ISelectedItemProps<T> extends IPickerItemProps<T> {
 }
 
 export type BaseSelectedItem = {
-  key?: React.Key;
+  key?: React.Key | null;
```

Comment:
This is to match an update to the type of React.Attributes to include null in the key